components/src/CommonMockHttpResponses/login_response.json

```diff
@@ -0,0 +1,8 @@
+{
+	"access_token": "fa702e32-c9b9-4b04-bdd4-8152f1fc1439",
+	"token_type": "bearer",
+	"expires_in": 604799,
+	"scope": "VESTRI_B2C",
+	"role": "PUBLIC",
+	"roles": ["PUBLIC"]
+}
```

Comment:
Please add new lines to each of your files

Reply:
@aChanEP This one is still missing a new line 😃

Reply:
kk updating